### PR TITLE
Added generated resource cleaning option to load_generator.

### PR DESCRIPTION
**Pull request by plomakin**
Implement SRC cloud cleaning with pre-generated objects from config.py

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by roman-verchikov** [comment]
You're repeating yourself a lot, but should work for now, +1

File: devlab/tests/generate_load.py [hunk 1]
```diff
@@ -146,11 +151,63 @@ def run_preparation_scenario(self):
...
         self.create_vms()
         self.create_vm_snapshots()
-        self.create_networks()
+
+    def clean_objects(self):
+        for tenant in config.tenants:
+            try:
+                self.keystoneclient.tenants.delete(
+                    self.get_tenant_id(tenant['name']))
+            except Exception:
```
Comment (miarmak) [inline devlab/tests/generate_load.py]: Maybe it makes sense to print original exception? Cause It can be hard to find the original reason of error.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by plomakin** [comment]
@miarmak these exceptions will be printed then: 
```
Tenant tenant1 failed to delete
list index out of range
User user1 failed to delete
list index out of range
User user2 failed to delete
list index out of range
VM server1 failed to delete
list index out of range
VM server2 failed to delete
list index out of range
VM server3 failed to delete
list index out of range
VM server4 failed to delete
list index out of range
VM server5 failed to delete
list index out of range
Network mynetwork1 failed to delete
list index out of range
```

Is it more helpful?

**Comment by miarmak** [comment]
I guess `print repr(e)` will be little bit informative. It will print exception name also as it's message.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by miarmak** [comment]
Thanks! +1

**Pull request merged**